### PR TITLE
Update emission related csv files for CRACMM2

### PR DESCRIPTION
**Pull request by tnskipper**
The following are updated:
* emissions/SPECIATEInputs/SPECIATEv5.2x_fromCRACMM.csv is updated to add a new column for the CRACMM2 species
* metadata/CRACMM1_2017_emiss_properties.csv is renamed to clarify that it is for CRACMM1
* emissions/cracmm_mapper.py is updated to handle missing SMILES in the SPECIATEv5.2x_fromCRACMM.csv file

### CODE REVIEW

**Review by havalapye — APPROVED** [review]

File: emissions/cracmm_mapper.py [hunk 1]
```diff
@@ -48,6 +48,12 @@ def get_cracmm_roc(smiles_input,koh,log10cstar,phase=None):
     # CRACMM2
     
     # Prep inputs
+    if smiles_input == '-':
+        unksmiles_msg = (
+            f'SMILES {smiles_input} not recognized. Mapping to UNKSMILES.'
+        )
+        warnings.warn(unksmiles_msg)
+        return 'UNKSMILES'
```
Comment (bnmurphy) [inline emissions/cracmm_mapper.py:56]: Good idea

**Comment by bnmurphy** [comment]
I noticed that the CRACMM2 names for the SVOC and LVOC ROC species are prepended with a V (e.g. line 2689 = VROCP0ALK). Should these start with A instead? I thought we decided to have the phase distinction done by the mapper. Or is this table for a more general case?

**Comment by havalapye** [comment]
> I noticed that the CRACMM2 names for the SVOC and LVOC ROC species are prepended with a V (e.g. line 2689 = VROCP0ALK). Should these start with A instead? I thought we decided to have the phase distinction done by the mapper. Or is this table for a more general case?

An optional argument has been added to the mapper (new for CRACMM2 version) that allows the user to specify the phase. If someone is running a VOC profile through the mapper, they may want to force everything to "V". This is how Karl currently uses it in S2S-Tool because S2S-Tool only uses the mapper for gas phase species. The mapper also only ever returns one mapping so it can't return a VROCP0ALK and AROCP0ALK. It also does not diagnose partitioning for semivolatiles. If the user wants things partitioned, they should do that upstream of the mapper and then call the mapper for each species + phase combination.

**Comment by bnmurphy** [comment]
Ah okay, I misunderstood this sentence in a previous email: "To get modulation between phases at a given loading the user would need to toggle that flag to what they want when they call the function."

I had thought that meant the mapper was going to be able to input a loading and then assign V or A to each species based on an input loading. 

Do I understand that the functionality is actually that the user would need to run the mapper in 'gas' mode and 'particle' mode and then cut and paste their map together by selecting manually which species mappings to take from each?

**Comment by tnskipper** [comment]
I'm not totally sure if this answers the question or not, but below is a simple example of how you could map emissions to V/A ROC species. Essentially, you would need to organize your input so that you have already separated the emissions of a particular species into the amounts that you want mapped to VROC and AROC and then also specify the phase to the mapper.

Code:
```
import pandas as pd
from cracmm_mapper import get_cracmm_roc

data = {
    'species':    ['SVOCP1 gas', 'SVOCP1 particle'],
    'emis':       [5, 15],
    'SMILES':     ['CCCCCCCCCCCCCCCCCCCCCCCCCCC', 'CCCCCCCCCCCCCCCCCCCCCCCCCCC'],
    'kOH':        [3.3800E-11, 3.3800E-11],
    'log10Cstar': [1., 1.],
    'phase':      ['gas', 'particle']
}
df = pd.DataFrame(data)
df['CRACMM2'] = df.apply(lambda x: get_cracmm_roc(x['SMILES'], x['kOH'], x['log10Cstar'], x['phase']), axis='columns')
df
```

Output:
image

**Review by bnmurphy — APPROVED** [review]

**Pull request merged**